client/App.js

```diff
@@ -49,51 +56,120 @@ class App extends React.Component {
         this.setState({productQuantity: productQuant-1})
     }
 
-    displayModal() {
-        console.log('DISPLAYING MODEL');
-        document.getElementById('myModal').style.display = 'flex'
+    displayModal(id) {
+        //console.log('DISPLAYING MODEL');
+        document.getElementById(id).style.display = 'flex'
     }
 
-    closeModal() {
-        console.log('CLOSING MODEL');
-        document.getElementById('myModal').style.display = 'none'
+    closeModal(id) {
+        //console.log('CLOSING MODEL');
+        document.getElementById(id).style.display = 'none'
     }
 
-    componentDidMount() {
-        axios.get('/displayProduct')
+    displayDeliveryDrawer() {
+        if (document.getElementById('b_deliveryHiddenDiv').style.display === 'none') {
+            document.getElementById('b_deliveryHiddenDiv').style.display = 'flex';
+        } else {
+            document.getElementById('b_deliveryHiddenDiv').style.display = 'none'
+        }
+    }
+
+    recordShoppingCartVal() {
+       //console.log(Number(document.getElementById('Quantity').innerText));
+       let numToAdd = Number(document.getElementById('b_Quantity').innerText);
+       this.setState({shoppingCartVal: this.state.shoppingCartVal + numToAdd});
+    }
+
+    onHoverColorChange(e) {
+        e.target.style.backgroundColor = 'lightgrey'
+    }
+
+    changeWidthOnStars() {
+        let newSizePercent = this.state.productReviewAvg * 20;
+        console.log(newSizePercent);
+        document.getElementById('b_starColorID').style.width = `${newSizePercent}%`
+    }
+
+    scrollToReviewDrawer() {
+        console.log(window.scrollY);
+        document.body.scrollTop = 1350
+    }
+
+    newReview(newReviewRating) {
+        let productCounter = this.state.productReviewCounter;
+        let totalReviewRating = this.state.productReviewAvg * this.state.productReviewCounter;
+        console.log(totalReviewRating);
+        this.setState({productReviewCounter: productCounter + 1}, () => {
```

Comment:
The formatting on this section is a bit hard to read, I'd expand out the set state and make it clear where the callback within the set state function is being started